acceptance/suites/tests/020-external-ca/020_retrieve_new_cert.rb

```diff
@@ -0,0 +1,16 @@
+test_name 'QA-1393 - C62572 - Register Agent with Puppet Master using an External CA'
+
+lockfile = '/opt/puppetlabs/puppet/cache/state/agent_catalog_run.lock'
```

Comment:
Seems okay, but maybe would be better to just stop the agent service so that it doesn't unintentionally run when we don't want it to.  Not sure how PE tests typically do this sort of thing...